css-box-model-challenge/css-box-model-challenge.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./css-box-model-challenge.css">
+</head>
+<body>
+    <header>
+        <h1>Welcome to my page</h1>
+        <p class="subtitle">This page lists my favourite bands!</p>
+    </header>
+    <main>
+        <h2>Favourite Bands:</h2>
+        <div class="bandsContainer">
+            <p class="bandListItem specialStyle">Dance Gavin Dance</p>
```

Comment:
Would kebab case be better here? I know in react we don't use kebab case but in raw html and css the convention is kebab case :)

Reply:
my bad. Will change it later. Thank you for the reminding:)